web.main.js

```diff
@@ -14,9 +14,14 @@ var minify = require('gulp-minify');
 var cleanCss = require('gulp-clean-css');
 const path = require('path');
 var pluginUtil = require('gulp-util');
+var zlib = require('zlib');
```

Comment:
zlib is already part of the dependencies?
Since we are at it, what about brotli? Did we add that dependency to the dev-tasks?

Reply:
zlib it's a module that comes with node.js and already implements brotli compression, so no need for additional dependencies, node.js already provides us with what we need.